zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -0,0 +1,180 @@
+
+        logger.debug("scan_vm_metadata on %s: found %d entries", metadataDir, len(entries))
+        return entries
+
+    def _do_cleanup(self, metadataPath):
+        with self._get_path_lock(metadataPath):
+            for path in [metadataPath, metadataPath + '.tmp',
+                         metadataPath + '.summary', metadataPath + '.summary.tmp']:
+                try:
+                    if os.path.exists(path):
+                        os.remove(path)
+                except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)